tm2/pkg/bft/consensus/state.go

```diff
@@ -27,8 +28,16 @@ import (
 	"github.com/gnolang/gno/tm2/pkg/service"
 	"github.com/gnolang/gno/tm2/pkg/telemetry"
 	"github.com/gnolang/gno/tm2/pkg/telemetry/metrics"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/codes"
+	"go.opentelemetry.io/otel/trace"
 )
 
+// -----------------------------------------------------------------------------
+// Tracer
+var tracer = otel.Tracer("consensus")
```

Comment:
We will likely implement this in other packages, so I think we should prefix the name of the tracer to provide more context.

```suggestion
var tracer = otel.Tracer("tm2/bft/consensus")
```